mmaction/datasets/pipelines/augmentations.py

```diff
@@ -1053,35 +1199,67 @@ class Flip:
 
     Reverse the order of elements in the given imgs with a specific direction.
     The shape of the imgs is preserved, but the elements are reordered.
-    Required keys are "imgs", "img_shape", "modality", added or modified
-    keys are "imgs", "lazy" and "flip_direction". Required keys in "lazy" is
-    None, added or modified key are "flip" and "flip_direction". The Flip
-    augmentation should be placed after any cropping / reshaping augmentations,
-    to make sure crop_quadruple is calculated properly.
+
+    Required keys are "img_shape", "modality", "imgs" (optional), "kp"
+    (optional), added or modified keys are "imgs", "kp", "lazy" and
+    "flip_direction". Required keys in "lazy" is None, added or modified key
+    are "flip" and "flip_direction". The Flip augmentation should be placed
+    after any cropping / reshaping augmentations, to make sure crop_quadruple
+    is calculated properly.
 
     Args:
         flip_ratio (float): Probability of implementing flip. Default: 0.5.
         direction (str): Flip imgs horizontally or vertically. Options are
             "horizontal" | "vertical". Default: "horizontal".
         flip_label_map (Dict[int, int] | None): Transform the label of the
             flipped image with the specific label. Default: None.
+        left (list[int]): Indexes of left keypoints, used to flip keypoints.
+            Default: [1, 3, 5, 7, 9, 11, 13, 15]. (COCO-17P keypoints)
+        right (list[ind]): Indexes of right keypoints, used to flip keypoints.
+            Default: [2, 4, 6, 8, 10, 12, 14, 16]. (COCO-17P keypoints)
         lazy (bool): Determine whether to apply lazy operation. Default: False.
     """
+    # Only horizontal flip is useful
     _directions = ['horizontal', 'vertical']
 
     def __init__(self,
                  flip_ratio=0.5,
                  direction='horizontal',
                  flip_label_map=None,
+                 left=[1, 3, 5, 7, 9, 11, 13, 15],
```

Comment:
Defaults these to None

Reply:
That's the default value for COCO-keypoint, set them as default values can save some text in config

Reply:
Clarity comes before saving space. The point is, these should be *explicitly* set  in configs. They are meaningless to most "Flip" operations. Their presence will make the img flip quite confusing.

Reply:
OK